pyproject.toml

```diff
@@ -533,6 +534,7 @@ alibaba = [
   "oss2>=2.14.0",
 ]
 amazon = [
+  "PyAthena>=3.0.10",
```

Comment:
BTW. It turned out that PyAthena dependency was also manuall added here, which is a bad idea, this part of pyproject.toml will be always now regenerated automatically when things will change.